src/app/app.module.ts

```diff
@@ -2,24 +2,31 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
-import {ApproutingModule} from './approuter.module';
-import {ApiService} from '../services/api';
-import {HttpClient, HttpClientModule} from '@angular/common/http';
-import {AuthService} from '../services/authservice/';
-import {FormsModule} from '@angular/forms';
-import {UserService} from '../services/userservice';
+import { ApproutingModule } from './approuter.module';
+import { ApiService } from '../services/api';
```

Comment:
import { ApiService } from '../services/api'; Consolidate imports to "index.ts" file and replace mentioned code to import { ApiService, AuthService, UserService, AuthGuardService } from '../services/';